test/categories/test_company.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from qual_id.categories.company import Company
+from test.utils.category_helper import CategoryHelper
+
+
+class TestCompany(unittest.TestCase):
+    def setUp(self):
+        self.company = Company()
+
+    def test__get_values__is_valid(self):
+        error_message = CategoryHelper.get_values_error_message(self.company)
+        self.assertTrue(error_message == "", error_message)
+
```

Comment:
Sorry @gabrielbarker , I want to ask, I've changed test_company to be like this, but it's still failed, guess what's wrong with this code?

Reply:
Hey @aldinaufal21 , I recently added some tests that make sure all the values in the `get_values` method are in alphabetical order. There are details of the recent changes in the `contributing.md` file :)